applications/odyssey_momentum.md

```diff
@@ -0,0 +1,373 @@
+# Odyssey - Momentum
+
+- **Team Name:** Odyssey B.V.
+- **Payment Address: 0x826fde427152d9fc0bf045dd858baa2de06b383b (USDC) 
+- **[Level](https://github.com/w3f/Grants-Program/tree/master#level_slider-levels):** 3
+
+## Project Overview :page_facing_up:
+
+### Overview
+
+#### Tagline: Staking in NFTs and Tokenize Momentum Plugins
+
+Odyssey is building Momentum, an open source, metaverse for digital societies. 
+
+The Momentum stack enables people to jointly build their own 3D worlds as well as develop and integrate plugins to unlock a wide range of decentralized 3D social action medium use cases. 
```

Comment:
We are basically a metaverse for co-creation and collaboration. A metaverse for activities rather than for content. People would come here for AMA’s, working together on complex issues, creating art together etc. 
As an example we can compare the virtual concert of Travis Scott in Fortnite with Momentum. Although the Travis Scott concert is EPIC in terms of size, it was a stream comparable to having many people watching the same video on YouTube. What we, in contrast to the Travis Scott example envision as a 3D social action medium, is a place where you can co-create making music together, playing in a band together in real-time and having presence in a 3D world while at the same time being miles apart. Hence a 3D social action medium.

Reply:
P.S. The deliverables will be adjusted end of day. I understand the issue. We will rephrase the deliverables, so they are independently verifiable.

Reply:
Deliverables adjusted to make them verifiable.